src/components/Icons/types.ts

```diff
@@ -14,6 +15,10 @@ export interface IconProps extends Omit<SvgProps, 'color'> {
   height?: number | string;
   /** A valid color for Astro's colors */
   color?: Colors[keyof Colors];
+  /** Size value */
```

Comment:
I think it would be nice to mention that this prop replaces the usage of widthand height;
```suggestion
  /** Size value. This value will be used as icon's width and height internally */
```